docs/sponsors/our-sponsors.md

```diff
@@ -4,7 +4,12 @@ description: PyTexas Conference 2024 Sponsors
 ---
 # PyTexas Conference 2024 Sponsors
 
-### Community Sponsors
+## Gold Sponsors
+[![Temporal Technologies Sponsor Logo](../assets/images/sponsors/temporal.png){: width="500" align="left"}](https://temporal.io)
+
+<br clear=all>
+
+#### Community Sponsors
```

Comment:
Why did the heading level change?

Reply:
Smaller tiers get smaller headings. Now that we have a gold tier it gets shifted.

Reply:
Oops. Also it removes them from the sidebar which I don't want. So ya. reverted this.